Library/Formula/dpkg.rb

```diff
@@ -1,9 +1,10 @@
 class Dpkg < Formula
   desc "Debian package management system"
   homepage "https://wiki.debian.org/Teams/Dpkg"
-  url "https://mirrors.ocf.berkeley.edu/debian/pool/main/d/dpkg/dpkg_1.18.3.tar.xz"
-  mirror "https://mirrorservice.org/sites/ftp.debian.org/debian/pool/main/d/dpkg/dpkg_1.18.3.tar.xz"
-  sha256 "a40ffe38d7f36d858a752189a306433cfc52c7d15d7b98f61d9f9dd49e0e4807"
+  url "http://http.debian.net/debian/pool/main/d/dpkg/dpkg_1.18.4.tar.xz"
```

Comment:
We don't adhere to using the same links as displayed on Debian's package URLs because there are so many secure mirrors that sync pretty much instantly with Debian. Canon doesn't really matter with Debian.